### PR TITLE
adding OTEL module 

### PR DESCRIPTION
**Pull request by SpeedyH30**
Here is a working build, though bash can be removed as it was added for testing. along with curl in the runDeps 
I don't think its the cleanest way of this working but nginx haven't made it very nice to work or add this module, but its something to work from. I've built it and had it pushing into a jaeger container, fancy index is still working along with other modules

### CODE REVIEW

**Comment by byjg** [comment]
Sorry for my delay in taking a look on it. I was trying to fix the build for forked repository. Now, the build is fixed. 

The only thing I am not confident in accept is the `conf/nginx.conf` change. It is too specific and could break other people's projects. 

I understand that is a nice to have, but that configuration can in a documentation. 

Also, you can inject this code by several ways:
* Build a docker image from the `nginx-extras` base and add this file
* Inject in runtime by defining a volume in the docker command line or docker compose
* If you are using Kubernetes you can inject in runtime by using a config map. 

So, if you remove the `conf/nginx.conf` and (optionally) add a documentation I can merge it right away and create a version tag for it.

**Comment by SpeedyH30** [comment]
Oh yeah not a problem. I had meant to comment out the additional config.
For show and completely forgot. I'll change that out in a mo

File: conf/nginx.conf
Comment (SpeedyH30) [inline conf/nginx.conf:1]: Ok this is now back to default

**Comment by byjg** [comment]
Release 1.26 published 

```
docker pull byjg/nginx-extras:1.26
```

Thank you.

**Comment by SpeedyH30** [comment]
Thanks very much

**Pull request merged**